src/style/TrackItemComponent.scss

```diff
@@ -125,4 +125,27 @@ a
     color: $color-text;
     // opacity: 0.7;
   }
-}
\ No newline at end of file
+}
+
+.kartrider-track-item-component.popup-component {
+  .border {
+    width: 300px;
+
+    &:hover {
```

Comment:
[Question]
I don't think popup UI might trigger `hover` event. I think it's useless. Could you remove this? What do you think?